pkg/testrig/template/main.go

```diff
@@ -0,0 +1,39 @@
+package template
+
+func GetMainStub() string {
+	return `package main
+
+import (
+	"context"
+	"log"
+	"net/http"
+
+	"github.com/go-chi/chi"
+
+	{{name}} "{{url}}"
+	{{impl.name}} "{{impl.url}}"
+)
+
+func LoadServices(ctx context.Context) error {
+	router := chi.NewRouter()
+
+	serviceInterface := {{impl.name}}.{{impl.interface_factory}}()
+
+	genCallbacks := {{impl.name}}.{{impl.callback_factory}}()
+
+	serviceHandler := {{name}}.NewServiceHandler(genCallbacks, &serviceInterface)
+
+	// Service Router
+	serviceRouter := {{name}}.NewServiceRouter(genCallbacks, serviceHandler)
+	serviceRouter.WireRoutes(ctx, router)
+
+	log.Println("starting {{name}} on :{{port}}")
+	log.Fatal(http.ListenAndServe(":{{port}}", router))
+	return nil
```

Comment:
return nil after a log.Fatal doesn't make much sense. Perhaps `return http.ListenAndServe(...)` and put the log.Fatal in main()?